src/liblibc/lib.rs

```diff
@@ -24,7 +24,7 @@
        html_playground_url = "https://play.rust-lang.org/",
        issue_tracker_base_url = "https://github.com/rust-lang/rust/issues/")]
 #![cfg_attr(test, feature(test))]
-#![feature(cfg_target_vendor)]
+#![cfg_attr(cfg_target_vendor, feature(cfg_target_vendor))]
```

Comment:
I think this should be `not(feature = "cargo-build")` instead of `cfg_target_vendor`, right? (e.g. similar to the above `feature` annotation)